### ISSUE TITLE
Calculate w1 parameter of weighted loss function

### ISSUE THREAD

**Issue by apyskir** [neptune-ml/open-solution-mapping-challenge#71]
Now it is just `torch.ones()`. It should handle class imbalance.

**Comment by kamil-kaczmarek** [comment]
No longer relevant. Closing this.
